src/sentry/auth/authenticators/u2f.py

```diff
@@ -31,6 +34,10 @@ class U2fInterface(AuthenticatorInterface):
         "Chrome)."
     )
     allow_multi_enrollment = True
+    # rp is a relying party for webauthn, this would be sentry.io on prod and the prefix for one's dev environment
+    rp_id = options.get("system.url-prefix").replace("https://", "")
```

Comment:
I think there may be a better way to get the domain. I might use [`urlib.parse`](https://docs.python.org/3/library/urllib.parse.html) to grab the host

Reply:
i'll look into it, i used this since i found other instances of this being used to get the url prefix

Reply:
oh interesting. Where did you see this being used?

The `url-prefix` could be `http`

Reply:
here are some cases that i found 
![image](https://user-images.githubusercontent.com/22259802/144487890-121ffdb6-6a95-41d5-8872-9cbf98f306ec.png)

Reply:
Oh those aren't replacing the protocol though right?

I was talking specifically about being careful replacing the protocol like that. It seems like you want just the `host` in your case righ?

Reply:
ah icic i misunderstood, it seems i can do this  `urlparse(options.get("system.url-prefix")).hostname`? or more specifically import `get_server_hostname()` from `/sentry/http.py` since it just returns the above

Reply:
`get_server_hostname` sounds great!

Reply:
hmm seems like theres an import problem so im gna use `urlparse(options.get("system.url-prefix")).hostname`

Reply:
What's the import error?

Reply:
seems like a circular import `ImportError: cannot import name 'EventError' from partially initialized module 'sentry.models' (most likely due to a circular import)`

Reply:
😢